src/Window.hpp

```diff
@@ -30,6 +30,14 @@ enum eGroupRules {
     GROUP_OVERRIDE    = 1 << 6, // Override other rules
 };
 
+enum eGetWindowProperties {
+    WINDOW_ONLY      = 0, // 0, 1, 2 are mutually exclusive
+    RESERVED_EXTENTS = 1,
```

Comment:
with bitfields please make all fields except 0 a `(1 << x)`

Reply:
since the beginning of the enum is mutually exclusive it doesn't make sense combining them
so we can use numbers that can't be represented as `1 << n`, for example 3

since they are less than `1 << 2` (1st element) they don't interfere with it

should I still use only `1 << n` for the enum's items?

Reply:
it's a stylistic choice, for consistency